File: PHP/uberPool.php
<?php
class UberPool extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $passengers, $brand, $model) {
        parent::__construct($license, $driver, $passengers);
        $this->brand = $brand;
        $this->model = $model;
    }
}

?>

File: Java/UberPool.java
package Java;

class UberPool extends Car {
    String brand;
    String model;

    public UberPool(int id, String license, Account driver, int passengers, String brand, String model) {
        super(id, license, driver, passengers);
        this.brand = brand;
        this.model = model;
    }
}